hack/tests-with-dind.sh

```diff
@@ -69,6 +69,20 @@ if [[ $(uname) == "Darwin" ]]; then
   metatron_cert_mnt="-v ${HOME}/.metatron:/mnt/metatron"
 fi
 
+docker run $TTYFLAG --privileged \
+  --cgroupns private \
+  -v ${PWD}:${PWD} \
+  -w ${PWD} \
+  --label "$run_id" \
+  --rm --name "$titus_agent_name" \
+  -d titusoss/titus-agent
+
+docker exec $TTYFLAG --privileged -e DIND=true -e DEBUG=${debug} -e SHORT_CIRCUIT_QUITELITE=true -e GOPATH=${GOPATH} "$titus_agent_name" \
```

Comment:
I don't understand, are we not already running these tests, but in a different context? Different github runner.
We probably don't need to run them twice, right?

Reply:
I don't claim to really understand the CI environment these are run in, but it looks like it doesn't have perms to create cgroups by default:

https://app.circleci.com/pipelines/github/Netflix/titus-executor/2135/workflows/bb67c865-5f66-439f-af0a-d2193f655898/jobs/20525#step-102-156

I skip the test in that case, but I added this bit so that they would at least run somewhere.